matlab/schemas/+vis/FancyBar.m

```diff
@@ -0,0 +1,20 @@
+ %{
+vis.FancyBar (manual) # pink noise with periods of motion and orientation$
```

Comment:
The dollar sign $ at the end is a bug from a really old version of MySQL.

The comment seems inaccurate.